tutorials/nextcloud-instantapp/index.mdx

```diff
@@ -22,49 +24,77 @@ In this tutorial we show you how to can deploy your personal cloud storage with
   - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key)
 </Message>
 
+## Creating the NextCloud Instance
+
 1. Click **Instances** in the **Compute** section of the side menu. The Instance creation page displays.
 2. Click **Create an Instance**. The Instance creation wizard displays.
-3. Choose the **Nextcloud** image in the **InstantApps** tab.
-4. Follow the remaining steps to [create your Instance](/compute/instances/how-to/create-an-instance/).
+3. Choose the Availability Zone for your Instance.
+4. Select the Instance type you want to create.
+5. Select the **NextCloud** image in the **InstantApps** tab.
+    <Lightbox src="scaleway-instantapp-nextcloud.webp" alt="" />
```

Comment:
Maybe the image could be dropped completely, since the other InstantApp tutorials do not show them. This would save the burden of maintaining the screenshots whenever the Scaleway Console appearance or the InstantApp list changes.

Reply:
Agreed :)